validator/js/nodejs/README.md

```diff
@@ -67,6 +67,12 @@ would allow applying the validator rules for `AMP4ADS` or `AMP4EMAIL` as well. `
 
 ## Release Notes
 
+### 1.0.35
+
+-   Set default engine to the WebAssembly version validator, which is
+    `https://cdn.ampproject.org/v0/validator_wasm.js`.
+    (ampproject/amphtml #34213)
```

Comment:
lets make this a link:
`[ampproject/amphtml #3213](https://github.com/ampproject/amphtml/pull/34213)`

Reply:
Done